teamB/shinInHuck/myapp/routes/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+const postRouter = require("./postRouter");
+const likeRouter = require("./likeRouter");
+router.use("/users", userRouter.router);
+router.use("/posts", postRouter.router);
+router.use("/likes", likeRouter.router);
```

Comment:
- 해당 부분은 가독성을 위해 아래와 같이 줄바꿈을 해주세요 ~!
```suggestion
const userRouter = require("./userRouter");
const postRouter = require("./postRouter");
const likeRouter = require("./likeRouter");

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
router.use("/likes", likeRouter.router);
```